user_data_notebooks/oco2-mip-co2budget-yeargrid-v1_User_Notebook.ipynb

```diff
@@ -42,34 +42,12 @@
    "metadata": {},
```

Comment:
<p>Modify this to not print every call of function. (it's a print line in a different block).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/US-GHG-Center/ghgc-docs/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>